apps/web/src/components/Menu/config/config.ts

```diff
@@ -180,6 +196,10 @@ const config: (
       icon: MoreIcon,
       hideSubNav: true,
       items: [
+        {
+          label: t('Buy Crypto'),
+          href: '/buy-crypto',
+        },
```

Comment:
Is this PM require?
show `Buy Crypto` in so many place.
Can we just show at `...` ?
Because we sure have a banner in home page tell user we have this new feature.

Reply:
resolved